### PR TITLE
Make sure malformed settings don't cause big problems

### PR DESCRIPTION
**Pull request by apple502j**
Since there are some people who want to load crafted settings, this PR makes sure they don't cause big problems (such as crashes).

### CODE REVIEW

**Comment by WorldLanguages** [comment]
Should we care about this? If users are able to change settings through the console then they likely know what they're doing?

**Comment by retronbv** [comment]
> Should we care about this? If users are able to change settings through the console then they likely know what they're doing?

They could also change it by importing malformed settings json

**Comment by WorldLanguages** [comment]
> They could also change it by importing malformed settings json

Good point, didn't think of that

File: addons/60fps/userscript.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ export default async function ({ addon, global, console }) {
         e.cancelBubble = true;
         e.preventDefault();
         mode = !mode;
-        if (mode) setFPS(addon.settings.get("framerate"));
+        if (mode) setFPS(Math.min(240, Math.max(31, addon.settings.get("framerate") | 31)));
```
Comment (GarboMuffin) [inline addons/60fps/userscript.js:24]: Surely that's not supposed to be a bitwise or?

Comment (apple502j) [reply addons/60fps/userscript.js:24]: whoops

**Comment by WorldLanguages** [comment]
Addons shouldn't validate settings

**Pull request closed without merging**